client/data/data-center/use-data-center-options.ts

```diff
@@ -0,0 +1,11 @@
+import { useTranslate } from 'i18n-calypso';
+
+export const useDataCenterOptions = () => {
+	const translate = useTranslate();
+	return {
+		bur: translate( 'US West (Burbank, California)' ),
+		dfw: translate( 'US Central (Dallas-Fort Worth, Texas)' ),
+		dca: translate( 'US East (Washington, D.C.)' ),
+		ams: translate( 'EU West (Amsterdam, Netherlands)' ),
```

Comment:
Same here, it could be replaced by a fetch to get available data centers.
fbhepr%2Skers%2Sjcpbz%2Sngbzvp.ncv%2Svapyhqrf%2Spynff%2Qngbzvp%2Qncv.cuc%3Se%3Qo6nn38q9%2377-og